app/views/routes_into_teaching/steps/completed.html.erb

```diff
@@ -0,0 +1,139 @@
+<%= render Content::LandingHeroComponent.new(
+  title: "Find your route into teaching",
+  colour: "pastel yellow-yellow",
+) %>
+
+<%= tag.div id: "routes_into_teaching_undergraduate_degree",
+  class: "hidden",
+  data: {
+    id: "undergraduate_degree",
+    key: session[:routes_into_teaching]["undergraduate_degree"],
+  }
+%>
+
+<%= tag.div id: "routes_into_teaching_unqualified_teacher",
+  class: "hidden",
+  data: {
+    id: "unqualified_teacher",
+    key: session[:routes_into_teaching]["unqualified_teacher"],
+  }
+%>
+
+<%= tag.div id: "routes_into_teaching_location",
+  class: "hidden",
+  data: {
+    id: "location",
+    key: session[:routes_into_teaching]["location"],
+  }
+%>
+
+<section class="container">
+  <div class="row inset">
+    <div class="col col-845">
+      <h2>You told us that you:</h2>
+
+      <ul>
+        <li><%= undergraduate_degree_summary %></li>
+        <li><%= unqualified_teacher_summary %></li>
+        <li><%= location_summary %></li>
+      </ul>
+
+      <% if non_uk? %>
```

Comment:
maybe not a question for the tech review - but is there a confusion between not living in England and being a non-UK citizen? E.g. should a UK citizen who lives in Scotland / Wales / NI be directed towards the iQTS route?
cc @gemmadallmandfe

Reply:
Good question - but yes, iQTS is based on location (outside England) and not citizenship

Reply:
thanks 👍